package.json

```diff
@@ -83,13 +83,15 @@
     "@types/semver": "^7.5.8",
     "@types/statuses": "^2.0.5",
     "@types/validator": "^13.12.2",
+    "@types/ws": "^8.5.14",
     "avsc": "^5.7.7",
     "aws-sdk": "^2.1692.0",
     "balena-device-config": "^6.3.0",
     "balena-semver": "^2.3.5",
     "basic-auth": "^2.0.1",
     "bluebird": "^3.7.2",
     "body-parser": "^1.20.3",
+    "bufferutil": "^4.0.9",
```

Comment:
Is this used anywhere?

Reply:
It's an opt-in for performance for ws: https://github.com/websockets/ws?tab=readme-ov-file#opt-in-for-performance